app/database/migrations/2018_07_03_155435_create_suggestion_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+/**
+ * This migration adds suggestion table to store suggestions of information of locations 
+ * generated by users.
+ */
+
+class CreateSuggestionTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('suggestion',function(Blueprint $table){
+            $table->increments('id');
+            $table->integer('user_id')->unsigned();
+            $table->foreign('user_id')->references('id')->on('user');
+            $table->integer('location_id')->unsigned();
+            $table->foreign('location_id')->references('id')->on('location');
+            $table->string('location_name', 255)->nullable();
+            $table->string('location_external_web_url', 255)->nullable();
+		    $table->string('location_address', 255)->nullable();
```

Comment:
We recently merged some Code Styling checks.  You can run these checks by getting latest and using:
composer cs
composer all

It looks like the indentation on line 29 is off and would cause the code styling check to fail.